src/modules/recipes/models.ts

```diff
@@ -0,0 +1,82 @@
+// Recipe için TypeScript türü
+export interface Recipe {
+  idMeal: string;
+  strMeal: string;
+  strMealThumb: string;
+  strCategory?: string;
+  strArea?: string;
+  strInstructions?: string;
+  strIngredient1?: string;
+  strIngredient2?: string;
+  strIngredient3?: string;
+  [key: string]: any; // Ek alanlar için
+}
+
+export type FormData = {
+  strMeal: string;
+  strCategory: string;
+  strArea: string;
+  strInstructions: string;
+  strMealImage: string;
+  ingredients: {
+    ingredient: string;
+    measure: string;
+    amount: string;
+  }[];
+};
+
+// RecipeDetail için TypeScript türü
+export interface RecipeDetail extends Recipe {
+  strInstructions: string;
+  ingredients: string[];
+}
+
+// ApiResponse için TypeScript türü
+export interface ApiResponse<T> {
+  meals: T[] | null;
+}
+
+// Category için TypeScript türü
+export interface Category {
```

Comment:
You can name this model CategoryDetails as Category name should be used for the enum holding all categories.

Reply:
I also saw a similar interface defined in the Categories component. The model from this file should be used instead, remove the local model.